src/rules/social/infoboard.js

```diff
@@ -0,0 +1,34 @@
+import { interval } from '../helpers';
+import { logger } from '../../logger';
+import { InfoEntry } from '../../models/infoentry';
+
+const POLL_FREQUENCY_MS = 10000;
+
+const closeInfoentryTimers = new Map();
+
+function closeInfoentry(infoentry) {
+	logger.info('Closing Infoentry', infoentry.get('id'));
+	return infoentry.save({ enabled: false }, { method: 'update', patch: true });
+}
+
+async function updateInfoentryScheduledToClose() {
+	const activeInfoentry = await new InfoEntry().where('enabled', true).where('active_until', '!=', '').fetchAll();
```

Comment:
Probably should check for non-null values instead of non-empty strings:
```suggestion
	const activeInfoentry = await new InfoEntry().where('enabled', true).whereNotNull('active_until').fetchAll();
```

Reply:
Tested, doesn't work. Crashes the backend.

```
        const activeInfoentry = await new InfoEntry().where('enabled', true).whe
reNotNull('active_until').fetchAll();
                                                                      ^
TypeError: (intermediate value).where(...).whereNotNull is not a function
    at updateInfoentryScheduledToClose (/workspace/src/rules/social/infoboard.js
:15:71)
```

Reply:
Yeah, was thinking of Knex syntax 😅 Probably should be `.where('active_until', '!=', null)`, but haven't tested it.

Reply:
In Postgres, `null is null` is `true` but `null = null` is `null`, which is falsy. However, also `null != null` and `null <> null` are `null`.

Usually one would use `x is null`/`x is not null` to check for nullity. If you need to treat `null` as a comparable value, the correct comparison operator is the monstrosity known as `is not distinct from`. https://www.postgresql.org/docs/current/functions-comparison.html

Haven't used this particular ORM but if you actually need a nullity comparison, my best guess would be `.where('active_until', 'is not', null)`.

That being said, if `active_until` has a default value that is the empty string `''`, then `.where('active_until', '!=', '')` is indeed the correct solution.